src/replay.rs

```diff
@@ -480,7 +341,7 @@ pub async fn replay(
         }
         if tr.id == txnid {
             if dump_mask & DUMP_ACCOUNT != 0 {
-                let path = format!("{}-{}.boc", state.account_addr.split(':').last().unwrap_or(""), txnid);
+                let path = format!("{}-{}.boc", state.account_addr, txnid);
```

Comment:
It's better to write addres without wc, because Winows can't save a file with ':" in the name